tests/test__routines.py

```diff
@@ -158,7 +177,13 @@ def test_routine_0(glue_database, glue_table, path, use_threads, concurrent_part
     assert comments["c2"] == "2!"
     assert comments["c3"] == "3"
 
-    # Round 6 - Overwrite Partitioned
+
+@pytest.mark.parametrize("use_threads", [True, False])
+@pytest.mark.parametrize("concurrent_partitioning", [True, False])
+@pytest.mark.parametrize("table_type", ["EXTERNAL_TABLE", "GOVERNED"])
+def test_routine_1(glue_database, glue_table, table_type, path, use_threads, concurrent_partitioning):
```

Comment:
`routine_0` had to be split in two. This is because "GOVERNED" tables do not allow to add partition keys once the table is already created